packages/grid_client/scripts/applications/static_website.ts

```diff
@@ -0,0 +1,121 @@
+import { FilterOptions, GatewayNameModel, MachinesModel } from "../../src";
+import { config, getClient } from "../client_loader";
+import { log, pingNodes } from "../utils";
+
+async function deploy(client, vms, subdomain, gatewayNode) {
```

Comment:
Let's define the type of the parameters please, and let's apply it to the whole script